data_registry/models.py

```diff
@@ -58,6 +58,10 @@ class Status(models.TextChoices):
         "<dd>The ID of the job in Scrapyd</dd>"
         "<dt><code>scrapy_log</code></dt>"
         "<dd>A local URL to the log file of the crawl in Scrapyd</dd>"
+        "<dt><code>item_dropped_count</code></dt>"
+        "<dd>The number of items dropped by the crawl</dd>"
+        "<dt><code>invalid_json_count</code></dt>"
+        "<dd>The number of invalid JSON files dropped by the crawl</dd>"
```

Comment:
In a JSONL file, I think it will drop only the invalid lines. Need to update migration as well.

```suggestion
        "<dd>The number of invalid JSON items dropped by the crawl</dd>"
```